libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1697,6 +1697,21 @@ bool AP_Arming::pre_arm_checks(bool report)
 bool AP_Arming::arm_checks(AP_Arming::Method method)
 {
 #if AP_RC_CHANNEL_ENABLED
+    if (method == AP_Arming::Method::RUDDER) {
```

Comment:
if ARMING_CHECK=0 then will that mean arming/disarming by rudder input will always be enabled? arm_checks() not called?

Reply:
Fixed and test added.